R/axis.R

```diff
@@ -68,28 +68,39 @@ axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL, ap
   
   fun.name <- "axis"
   
-  user_args <- filter_arguments(fun.name = fun.name, ..., 
-                                custom.config = object[["global"]][["config"]][["config.file"]])$call.args
+  args <- filter_arguments(fun.name = fun.name, ..., 
+                           custom.config = object[["global"]][["config"]][["config.file"]])
   
-  sides <- user_args[[fun.name]]$side
-  user_args[[fun.name]]$side <- NULL
-  user_args[[fun.name]]$n.minor <- n.minor
-  user_args[[fun.name]]$tcl.minor <- tcl.minor
+  if (length(args$extracted.args) > 0){
+    warning('axs.gsplot is not configured to support embedded function calls', call. = FALSE)
+  }
+  
+  user.args <- args$call.args
+  sides <- user.args[[fun.name]]$side
+  
+  if (is.null(sides)){
+    return(object)
+  }
+  
+  user.args[[fun.name]]$side <- NULL
+  user.args[[fun.name]]$n.minor <- n.minor
+  user.args[[fun.name]]$tcl.minor <- tcl.minor
+  
+  user.args[[fun.name]] <- append_replace(user.args[[fun.name]], args$option.args)
```

Comment:
correct me if I'm wrong, but this is the legitimate change for this PR, right? The others are adding warnings, catching nulls, and changing code style/object names, but this line is what has been actually added